Google.PowerShell.IntegrationTests/Container/Container.GkeCluster.Tests.ps1

```diff
@@ -162,10 +184,151 @@ Describe "New-GkeNodeConfig" {
     It "should raise an error for wrong DiskSize" {
         { New-GkeNodeConfig -DiskSizeGb 3 } | Should Throw "less than the minimum allowed range of 10"
     }
+}
+
+Describe "Add-GkeCluster" {
+    $r = Get-Random
+
+    # Create the cluster in parallel to reduce wait time.
```

Comment:
I refactored the code a little. It takes about 6 minutes to create a cluster so for 4 clusters, that would be 24 minutes. If we create them with jobs, however, it will only be about 7 or 8 minutes for all 4.